packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/sections-three-up~compact.json

```
@@ -0,0 +1,53 @@
+{
+  "sectionThreeUp" : {
+    "compact": "True",
```

```suggestion
    "compact": "True",
```
I think we should use boolean `true` instead of a string for the value
Please update all the boolean values in this PR

---

changed this value